src/components/ValidateSubscriber/ValidateMaxSubscribersForm.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+import { Field, Form, Formik, FormikState } from "formik";
+import { FormattedMessage } from "react-intl";
+import { CheckboxFieldItem, FieldGroup, InputFieldItem } from "./form-helpers";
+import {
+  MaxSubscribersQuestion,
+  SubscriberValidationAnswer,
+  ValidateMaxSubscribersFormProp,
+} from "./types";
+
+export const ValidateMaxSubscribersForm = ({
+  validationFormData,
+  handleClose,
+  handleSubmit,
+}: ValidateMaxSubscribersFormProp) => {
+  // TODO: validate and onSubmit implementation
+  const validate = (values: {}) => {};
+  const onSubmit = async (values: {}) => {};
+
+  const [show, setShow] = useState(false);
+  const toggleOthers = () => {
+    console.log("test toggle");
+    setShow((prev) => !prev);
+  };
+
+  const isCheckbox = (answer: SubscriberValidationAnswer) => {
+    return ["CHECKBOX_WITH_TEXTAREA", "CHECKBOX"].includes(answer.answerType);
+  };
+
+  const isCheckboxWithTextArea = (answer: SubscriberValidationAnswer) => {
+    return "CHECKBOX_WITH_TEXTAREA" === answer.answerType;
+  };
+
+  const initialValues = validationFormData.questionsList?.reduce(
+    (accumulator, question, index) => {
+      const answer = {
+        [`answer${index}`]: isCheckbox(question.answer) ? [] : "",
+        ...(isCheckboxWithTextArea(question.answer) && {
+          [`answer${index}_text`]: "",
+        }),
+      };
+
+      return { ...accumulator, ...answer };
+    },
+    {}
+  );
+
+  const renderQuestions = (
+    questionItem: MaxSubscribersQuestion,
+    questionIndex: number,
+    formikProps: FormikState<any>
+  ) => {
+    const { touched, errors, submitCount } = formikProps;
+    if (
+      questionItem.answer?.answerType === "TEXTFIELD" ||
+      questionItem.answer?.answerType === "URL"
+    ) {
+      return (
+        <InputFieldItem
+          type="text"
+          label={questionItem.question}
+          fieldName={`answer${questionIndex}`}
+          required
+          className={`${
+            questionItem.answer?.answerType === "TEXTFIELD"
+              ? "field-item--50"
+              : ""
+          }`}
+        />
+      );
+    }
+
+    if (isCheckbox(questionItem.answer)) {
+      let fieldName = `answer${questionIndex}`;
```

Comment:
Muy menor, pero ya que no se muta, podríamos usar `const`?

```suggestion
      const fieldName = `answer${questionIndex}`;
```